src/shared-components/aboutUsCard/aboutUsCard.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import LinkButton from "../linkButton/linkButton";
+import { useHistory } from "react-router-dom";
+import "./aboutUsCard.css";
+
+const AboutUsCard = (props) => {
+  let history = useHistory();
+
+  function handleClick() {
+    history.push("/about-us");
+  }
+  return (
+    <>
+      <div className="about-us-box">
+        <h6 className="mb-2 font-weight-bold">About Us</h6>
+        <p className="mb-3">
+          Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do
```

Comment:
@sangeetasg we should be replacing this dummy text right?

Reply:
@swapnilchincholkar Yes Sir